content/help/integrations/wordpress/using-comments.md

```diff
@@ -1,23 +0,0 @@
----
-title: Using Comments
-tags:
-- Features
-- Embedding
-category: help
-menu:
-  help:
-    parent: wordpress
-aliases:
-- /features/wordpress/using-comments
```

Comment:
Missing alias, you also need `/help/integrations/wordpress/using-comments/`

Reply:
Fixed